packages/atomic/src/components/common/button.tsx

```diff
@@ -22,6 +22,7 @@ export interface ButtonProps {
   ariaChecked?: string;
   ariaCurrent?: string;
   ariaControls?: string;
+  ariaHidden?: string;
```

Comment:
`ariaHidden` should be a boolean here.

Reply:
ariaExpanded?: string;
  ariaPressed?: string;

are all strings, and I think it's a string attribute? e.g. `true.toString()`

Reply:
ah, I believe they should've been booleans in the first place.

And yes, the actual attribute is a string.

Reply:
we can change all of them then in a separate PR, but I'll put consistency first here

Reply:
ok, I'll keep it a string then !